tests/nested/core/core-fde-state/task.yaml

```diff
@@ -0,0 +1,27 @@
+summary: Check the FDE state is correctly initialized on first boot
+
+details: |
+    This test checks checks that the FDE state is correctly
+    initialized on first boot.
+
+systems: [ubuntu-20.04-64, ubuntu-22.04-64, ubuntu-24.04-64]
+
+execute: |
+    echo "Wait for the system to be seeded first"
+    remote.exec "sudo snap wait system seed.loaded"
+
+    remote.exec "sudo cat /var/lib/snapd/state.json" >state.json
+
+    # We should probably check the digest matches the primary key, but
```

Comment:
that's `keyctl id ubuntu-fde:/dev/disk/by-uuid/2bc53768-8e9a-4d0d-86ff-8cf1dd2826a4:unlock:` away AFAIU, but we don't have keyctl to access the keyring, unless we add a debug tool that does what we want without introducing additional dependencies